src/pages/Home/Menu/MenuModalStyle.ts

```diff
@@ -0,0 +1,30 @@
+import styled from "styled-components";
+
+export const MenuModalStyle = styled.div`
```

Comment:
There should be a transition when it is opened and closed

Reply:
> There should be a transition